core/scss/components/sidebar.scss

```diff
@@ -0,0 +1,375 @@
+                  &:after {
+                    position: absolute;
+                    content: '';
+                    height: calc(100% - 33%);
+                    width: 2px;
+                    box-shadow: -12px 0 0 0 $blue-dark-press;
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+
+    &--dark.#{$prefix}-sidebar {
+      background-color: $gray-1;
+      .#{$prefix}-sidebar {
+
+        &-nav {
+          &__group {
+            &--primary {
+              .cui-list-item {
+                @include list-item-color(
+                  $active-bgrd-color: rgba(0, 115, 149, 0.56),
+                  $active-color: $white,
+                  $active-section-left-color: $white,
+                  $active-section-right-color: $white,
+                  $background-color: transparent,
+                  $color: $white,
+                  $focus-color: $white,
+                  $header-color: $white,
+                  $hover-color: $white,
+                  $hover-bgrd-color: rgba(0, 115, 149, 0.4),
+                  $includeFocus: false,
+                  $section-left-color: $white,
+                  $section-right-color: $white,
+                  $subheader-color: $white,
+                ); 
+              }
+
+            & .#{$prefix}-sidebar-nav__group--secondary {
+              .cui-list-item {              
+                @include list-item-color(
+                  $active-bgrd-color: rgba(0, 115, 149, 0.56),
+                  $active-color: $white,
+                  $active-section-left-color: $white,
+                  $active-section-right-color: $white,
+                  $background-color: transparent,
+                  $color: $white-80,
+                  $focus-color: $white-80,
+                  $header-color: $white-80,
+                  $hover-color: $white-80,
+                  $hover-bgrd-color: rgba(0, 115, 149, 0.4),
+                  $includeFocus: false,
+                  $section-left-color: $white,
+                  $section-right-color: $white,
+                  $subheader-color: $white-80,
+                ); 
+              }
+
+              & .#{$prefix}-sidebar-nav__group--tertiary {    
+                .cui-list-item {
+                  @include list-item-color(
+                    $active-bgrd-color: transparent,
+                    $active-color: $blue-press,
+                    $background-color: transparent,
+                    $color: $white-80,
+                    $focus-color: $blue-dark-hover,
+                    $hover-color: $blue-dark-hover,
+                    $hover-bgrd-color: transparent,
+                    $includeFocus: false,
+                  );
+      
+                    &:before {
+                      box-shadow: -12px 0 0 0 $white-12;
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+}
```

Comment:
Asking for a friend, is it normal to need to add all this scss or should this be in the core project?

Reply:
Good question, now that we're in a monorepo this is actually all in the core project.  What we're getting rid of us the side-nav file eventually and using only the sidebar.